src/effects/VST/VSTEffect.cpp

```diff
@@ -3790,9 +3790,15 @@ void VSTEffectWrapper::SizeWindow(int w, int h)
 
 void VSTEffectWrapper::Automate(int index, float value)
 {
+   // This is now called only for the instance;
+   // TODO (?) we'd need to copy here part of what is done in VSTEffectValidator::Automate
+   // - we need to send back to the framework the new value for the parameter
```

Comment:
The commit that adds this comment is making a promise to fix things in a later commit.  I'm still reviewing to see whether that happens.

Reply:
ok, done now